service/src/main/java/org/exoplatform/extension/exchange/service/util/CalendarConverterUtils.java

```diff
@@ -233,7 +235,8 @@ public static void convertExchangeToExoMasterRecurringCalendarEvent(CalendarEven
         dayOfMonth = recurrence.getStartDate().getDate();
       }
       event.setRepeatByMonthDay(new long[] { dayOfMonth });
-      Utils.adaptRepeatRule(event, DEFAULT_TIME_ZONE , CalendarService.PERSISTED_TIMEZONE);
+      CalendarSetting calendarSetting = getCalendarService().getCalendarSetting(username);
+      Utils.adaptRepeatRule(event, DateUtils.getTimeZone(calendarSetting.getTimeZone()), CalendarService.PERSISTED_TIMEZONE);

```

Comment:
Same comment as previous for multiple lines instruction

Reply:
Fixed.